config/crd/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 # This kustomization.yaml is not intended to be run by itself,
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
-resources:
-- mcad
```

Comment:
Should `mcad.yaml` be added there?

Reply:
i probably will take  a different approach.
Have 3 CRD yaml files in this `config/crd` , not combined all into 1 mcad.yaml
it is easier to compare if upstream has CRD changes and only apply changed yaml.

Reply:
I agree with @zdtsw that it'd be ideally to have them separated into different files.